### PR TITLE
Test users in probo script

### PR DESCRIPTION
**Pull request by dafeder**
Add test users for basic DKAN roles to probo sites

### CODE REVIEW

**Comment by dafeder** [comment]
Closing for now...

**Comment by dafeder** [comment]
Working

File: dkan-init.sh [hunk 1]
```diff
@@ -167,6 +167,15 @@ install_dkan() {
   fi
 }
 
+create_test_users() {
```
Comment (acouch) [inline dkan-init.sh]: I think this would make more sense in ``site.ahoy.yml``

**Comment by dafeder** [comment]
See latest commit @acouch

**Pull request merged**